src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ConclaveService.java

```diff
@@ -22,6 +24,16 @@
   private final WebClient conclaveWebClient;
   private final WebclientWrapper webclientWrapper;
 
+  @Value
+  @Builder
+  public static class UserContactPoints {
+
+    String phone;
+    String fax;
+    String web;
+    String email;
+  }

```

Comment:
Guess this class is a convenience over returning `ContactResponseInfo` objects - simpler for clients to extract the relevant data, and defined here as it's not part of the generated Conclave classes? (all fine - just checking I understand :))

Reply:
Yeah - I think that was the idea... 🤔 😄